lib/puppet/defaults.rb

```diff
@@ -306,7 +306,7 @@ def self.default_vendormoduledir
         :default  => ! Puppet::Util::Platform.windows?,
         :type     => :boolean,
         :desc     => "Whether Puppet should manage the owner, group, and mode of files it uses internally.
-        
+
```

Comment:
We might not actually want to remove the whitespace here because of our markdown formatting and how the text gets transformed during our docs publishing processes.